addons/website_event/models/event_event.py

```diff
@@ -233,7 +233,7 @@ def _compute_website_url(self):
         super(Event, self)._compute_website_url()
         for event in self:
             if event.id:  # avoid to perform a slug on a not yet saved record in case of an onchange.
-                event.website_url = '/event/%s' % slug(event)
+                event.website_url = '%s/event/%s' % (event.website_id.get_base_url(), slug(event))
```

Comment:
Actually you can just do `get_base_url()` and it will get the website_id url by default, if the field exists
And this way if it's overriden for events for some reason (e.g. custom modules), we actually use the override

Could switch to fstrings while we're here too.

Reply:
Agree, thanks!